tribe/events/v2/list.php

```diff
@@ -41,6 +41,7 @@
 
 		<?php $this->template( 'components/before' ); ?>
 
+		<h1>Events</h1>
```

Comment:
Just to bring in line with nhsuk-frontend and WP generics, can this be modified to:
`<h1 class="entry-title nhsuk-heading-l">` - this then allows for easier global changes to apply to all elements.

Reply:
Good point. Done.